users/vieting/experiments/switchboard/hybrid/feat/experiments.py

```diff
@@ -192,7 +192,11 @@ def run_baseline_scf():
     nn_args = get_nn_args_baseline(
         nn_base_args={
             "scf": dict(
-                returnn_args=dict(batch_size=14000),
```

Comment:
I did run this setup successfully with batch size 14k, so please do not change that. You can import this job from my setup.